File: 단계별로 풀어보기/Step 06/2562.cpp
//2562 최댓값
#include <iostream>
using namespace std;

int main() {
    int array[9];
    int num, max, idx;
    for(int i=0; i<9; i++) {
        cin >> num;
        array[i] = num;
    }
    max = array[0];

    for(int i=0; i<9; i++) {
        if(max <= array[i]) {
            max = array[i];
            idx = i+1;
        }
    }
    cout << max << endl << idx;
    return 0;
}

File: StepByStep/Python/Step_04_1차원 배열/2562.py
nums = []
for i in range(9):
    nums.append(int(input()))

maxNum = nums[0]
idx = 0

for j in range(9):
    if maxNum < nums[j]:
        maxNum = nums[j]
        idx = j

print(maxNum)
print(idx+1)

# print(max(nums))
# print(nums.index(max(nums))+1)
